process/software-project-lifecycle.md

```diff
@@ -0,0 +1,161 @@
+          - [Project Graduation Process: Graduated to Archiving](#project-graduation-process-graduated-to-archiving)
+          - [Submission Forms for Graduated to Archived](#submission-forms-for-graduated-to-archived)
+      - [Archived](#archived)
+
+<!-- /TOC -->
+
+
+## Stages - Definitions & Expectations
+
+Software focused projects have a maturity level of sandbox, incubating, or graduated. Archived is for such projects no longer in active development. The maturity level is a signal by OpenSSF as to what sorts of enterprises should be adopting different projects. Software focused projects increase their maturity by demonstrating their sustainability to OpenSSF’s Technical Advisory Committee(TAC): that they have adoption, a healthy rate of changes, committers from multiple organizations, have adopted the OpenSSF Code of Conduct, and have achieved and maintained the Core Infrastructure Initiative Best Practices Badge.
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage software projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level project
+* Facilitate alignment with existing Working Groups or Projects if (and only if) this is desired
+* Nurture projects (e.g. via OpenSSF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct and IP Policy requirements.
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Sandbox Entry Requirements
+
+Software focused projects being submitted to the OpenSSF at the sandbox level are intended to be the entry point for early stage projects and are not required to undergo due diligence.
+
+Sandbox projects should be early-stage projects that the OpenSSF TAC or parent Working Group believes warrant experimentation.
+
+* New software focused projects that are designed to extend one or more OpenSSF Projects with functionality or interoperability libraries.
+* Independent software focused projects that fit the OpenSSF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+* Software focused projects commissioned or sanctioned by the OpenSSF, including initial code for OpenSSF WG collaborations, and "experimental" projects
+* Any software focused project that realistically intends to join OpenSSF Incubation in future and wishes to lay the foundations for that.
+
+#### Project Graduation Process: Sandbox to Incubating
+
+Incubating software focused projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC or parent WG sponsor, with two weeks for public comment before a vote is called.
```

Comment:
akin to my comment on #95, even sandbox projects require legal review / IP transfer to be accepted. Further diligence should be reserved for Incubating, as here described.

Reply:
Should this depend on the state of project, like early stage projects with few mvp scripts shouldnt need legal review.

Reply:
No matter how "small" the codebase is, if there is a transfer of IP, then a legal sign-off is in everyone's best interest.

The deciding factor, in my past experience, has been whether the project is _started_ in the open (e.g., members of the foundation spin up an empty repo and start working together) or whether the project was _started_ in private and then contributed. Even "just a couple scripts" can be associated with a patent claim or a trademark. This is a legal bar, not a technical bar, and the TAC is not the deciding body on what requires legal review.